client/src/components/FlightDetails.vue

```diff
@@ -106,13 +106,22 @@
       <button v-if="showEditButton" class="btn btn-outline-primary btn-sm ms-2">
         <i class="bi bi-pencil-square mx-1"></i>Flug bearbeiten
       </button>
+
       <button
         v-if="!showEditButton && showAdminEditButton"
         class="btn btn-outline-danger btn-sm ms-2"
       >
         <i class="bi bi-pencil-square mx-1"></i>Admin
       </button>
     </router-link>
+    <button
+      v-if="!showEditButton && showAdminEditButton"
+      class="btn btn-outline-danger btn-sm ms-2"
+      @click="onRedoCalculation"
+    >
+      <i class="bi bi-calculator mx-1"></i>Neuberechnen
```

Comment:
Das finde ich etwas dick auftragend. Wir wollten doch alle Adminfunktionen in in Dropdown packen.